test/powershell/engine/PropertyAccessor.Tests.ps1

```diff
@@ -0,0 +1,96 @@
+#
+# Functional tests to verify basic conditions for IO to the PowerShellProperties.json files
+# The properties files are supported on non-Windows OSes, but the tests are specific to 
+# Windows so that file IO can be verified using supported cmdlets.
+# 
+
+try {
+
+    #skip all tests on non-windows platform
+    $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+    $IsNotSkipped = ($IsWindows -eq $true);
```

Comment:
Maybe `$IsNotSkipped = $IsWindows`?
Or even better, why not just use `$IsWindows` without `$IsNotSkipped`? I don't feel this variable helps readability.